### PR TITLE
Feat: Update candid file for NNS

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Governance is adding a new Action variant.

This PR: Updates candid file for NNS canisters.

# Changes

* Update candid files and related.
* Add the new variant in the Action converters, yet, not converting it. This will be done in another PR. This is ok because this proposal won't be used in a month or so.

# Tests

I tested that this version works against an older version of the Governance canister.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @peterpeterparker please review

**Review by mstrasinskis — APPROVED** [review]
Thanks, LGTM! Does the aggregator needs to be updated too?

File: packages/nns/src/canisters/governance/response.converters.ts [hunk 1]
```diff
@@ -392,6 +392,11 @@ const toAction = (action: RawAction): Action => {
     };
   }
 
+  if ("CreateServiceNervousSystem" in action) {
```
Comment (dskloetd) [inline packages/nns/src/canisters/governance/response.converters.ts:395]: Same question

File: packages/nns/src/types/governance_converters.ts [hunk 2]
```diff
@@ -21,6 +22,8 @@ export type Action =
   | {
       ExecuteNnsFunction: ExecuteNnsFunction;
     }
+  // TODO: Create new CreateServiceNervousSystem type (don't use array for optional fields)
+  | { CreateServiceNervousSystem: CreateServiceNervousSystem }
```
Comment (dskloetd) [inline packages/nns/src/types/governance_converters.ts:26]: I guess this is actually needed to satisfy the type system?

File: packages/nns/src/canisters/governance/request.converters.ts [hunk 3]
```diff
@@ -257,6 +257,11 @@ const fromAction = (action: Action): RawAction => {
     };
   }
 
+  if ("CreateServiceNervousSystem" in action) {
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply packages/nns/src/canisters/governance/request.converters.ts:260]: This is necessary to satisfy the type system indeed.

File: packages/nns/src/canisters/governance/request.converters.ts
[the remark this answers is not in the document]
Comment (lmuntaner) [reply packages/nns/src/canisters/governance/request.converters.ts:1]: Yes, that's correct.

I'll ask him to review the candid file changes, good point.

[the remark this answers is not in the document]
Comment (dskloetd) [reply packages/nns/src/canisters/governance/request.converters.ts:1]: Thanks. If he sees no issues, and no further changes are necessary, feel free to go ahead without my approval.

**Pull request merged**